docker-app/qfieldcloud/core/pagination.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import pagination, response
+
+
+class LimitOffsetPaginationResults(pagination.LimitOffsetPagination):
```

Comment:
```suggestion
class LimitOffsetPagination(pagination.LimitOffsetPagination):
```

I would prefer a bit shorter name. We have to write it a few dozen times :)

Reply:
@why-not-try-calmer would you mind if you implement the proposed naming? I would have expected to either comment on the suggestion if you don't agree or accomodate it :)

Reply:
In https://github.com/opengisch/qfieldcloud/pull/710/commits/f0eb815d8bc6fe646f9a99b0255ec6fe87ebf0e2 I accomodated the suggestion of renaming, but not with the particular name suggested. I have no preference about the name, but I think its unusual and a little bit misleading to give it the same name as its super-class. It's unusual in general to name classes after their super-classes, and it's misleading in this case because it really behaves differently.

Reply:
Thanks for the feedback. Yes, I overlooked the name matching.

`QfcLimitOffsetPagination` is a good name in this case, we have used similar patterns around the codebase.